lib/led_manager/LedManager.cpp

```diff
@@ -86,9 +86,19 @@ void LedManager::RunEffect() {
         virtual_index = strip_index;
       }
 
-      CRGB rgb = GetCurrentEffect()->GetRGB(virtual_index,
-                                            radio_state->GetNetworkMillis(),
-                                            strip, radio_state->GetSetEffect());
+      CRGB rgb;
+      if (!strip.FlagEnabled(Off)) {
```

Comment:
nit: prefer to use positive (un-negated) conditions for if - `if (strip.FlagEnabled(Off))`.